src/goto-cc/goto_cc_cmdline.cpp

```diff
@@ -117,3 +117,13 @@ void goto_cc_cmdlinet::add_infile_arg(const std::string &arg)
     fclose(tmp);
   }
 }
+
+bool goto_cc_cmdlinet::have_infile_arg() const
```

Comment:
I think we should probably just make sure to put definitions in cpp files in general.

Reply:
:+1: